src/tests/Microsoft.PowerFx.Core.Tests.Shared/UDFHasSameDefinitionTests.cs

```diff
@@ -0,0 +1,135 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.PowerFx.Core.Functions;
+using Microsoft.PowerFx.Core.Tests.Helpers;
+using Microsoft.PowerFx.Core.Texl;
+using Microsoft.PowerFx.Core.Utils;
+using Microsoft.PowerFx.Syntax;
+using Microsoft.PowerFx.Types;
+using Xunit;
+
+namespace Microsoft.PowerFx.Core.Tests
+{
+    public class UDFHasSameDefinitionTests : PowerFxTest
+    {
+        private static readonly ReadOnlySymbolTable _primitiveTypes = ReadOnlySymbolTable.PrimitiveTypesTableInstance;
+
+        [Theory]
+        [InlineData("Foo(x: Number): Number = Abs(x);", "Foo(x: Number): Number = Abs(x);", true)]
```

Comment:
>[InlineData("Foo(x: Number): Number = Abs(x);", "Foo(x: Number): Number = Abs(x);", true)] [](http://example.com/codeflow?start=8&length=90)

Can you add a trivial equal case for imperative UDFs?

Reply:
done